src/mantarray_desktop_app/mc_simulator.py

```diff
@@ -61,6 +73,22 @@ class MantarrayMcSimulator(InfiniteProcess):
         read_timeout_seconds: number of seconds to wait until read is of desired size before returning how ever many bytes have been read. Timeout should be set to 0 except in unit testing scenarios where necessary
     """
 
+    default_mantarray_nickname = "Mantarray Simulator (MCU)"
+    default_mantarray_serial_number = "M02001901"
+    default_pcb_serial_number = (
+        "TBD"  # TODO Tanner (3/17/21): implement this once the format is determined
+    )
+    default_firmware_version = "0.0.0"
+    default_metadata_values: Dict[UUID, Any] = {
```

Comment:
The UUID/value version is useful for testing and the bytes/bytes version makes including metadata in data packets more efficient. I agree it should also be an immutabledict, and I left the type as Any since I didn't know how many values would be added. It can probably also just be changed to Union[str, int].